strings/piglatinGame.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Arrays;
+public class piglatinGame 
+{
+    public static void main(String[] args) 
+    {
+         try 
+         {
+             if (args[0]  == null )

```

Comment:
Use scanner or bufferedReader class instead of {args} command line input may not be given, and directly function may be called.